www/creating-zooming-images.html

```diff
@@ -191,6 +196,44 @@ <h2>Creating Zooming Images</h2>
         <td>Free desktop app for locally viewing DZI files for Windows. Linux soon.</td>
         <td><a href="/examples/tilesource-dzi/">DZI</a></td>
     </tr>
+    
+    <tr>
+        <td colspan="3"><h4>Single-file Tile formats - see complete <a href="https://github.com/openseadragon/openseadragon/issues/944">discussion thread</a>. Useful for faster/easier file operations and maintenance on S3/file servers FS.</h4></td>
```

Comment:
Let's have just "Single-File Tile Formats" be the `h4` and the rest be normal text below it in the same `td`.